generic-styles/content-baked.less

```diff
@@ -26,6 +26,10 @@ ul, ol {
   color: @gray;
 }
 
+ol li {
+  margin-top: 1rem;
+}
```

Comment:
This will add also margin-top to the first item which will result in kind of margin-top: 2rem for `ol`
It will also add this margin to any nested `li` - I think you should ask someone to specify behavior for this